.env.test

```diff
@@ -0,0 +1,6 @@
+# Service dependencies
+NEO4J_PORT=7575
+
+# Federation
+LOCAL_DOMAIN=cb6e6126.ngrok.io
```

Comment:
Yeah the reason for this line is that when I wrote the tests, I basically recorded the interactions between quitter.no and my own server running from localhost through ngrok.io.